dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/inbound/HttpInboundServiceContextImpl.java

```diff
@@ -1216,7 +1216,10 @@ protected boolean check100Continue() {
         // if the channel stopped while we parsed this Expect request, we want
         // to send an error to close the connection and avoid the body transfer
         // PK12235, check for a full stop only
-        if (this.myLink.getChannel().isStopped()) {
+        // PH41928 check also for stopping. WC will reject this with 503 if the server
+        // is stopping.  cfw should send the 503 at this point so the body of the
+        // request is not lost.
+        if (!this.myLink.getChannel().isRunning()) {
             if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
                 Tr.debug(tc, "Channel stopped, sending error instead of 100-continue");
```

Comment:
What do you think about changing this debug to reflect the proper state, stopped|stopping, either which will be possible now? Perhaps something like: 

`"Channel" + this.myLink.getChannel().isStopped() ? "stopped": "is stopping" + ", sending ..." `

Reply:
what if "stopped"?  will this change also stop 100-continue?

Reply:
There are 2 states, running, stopping and stopped.  We want to issue the 503 here if either stopped or stopping, so I consolidated to be !isRunning().

Reply:
I guess the answer is yes :-) , it will send 503 if either stopped or stopping.

Reply:
Yes, I can update the trace!

Reply:
Trace comment has been udpated